src/js/components/common/Link.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+import React from "react"
+
+import {shell} from "electron"
+
+type Props = {|href: string, children: *|}
+
+export default function Link({href, children}: Props) {
```

Comment:
Nice, a new common component! I had also included these in the common/index.js so that we can import/export the same way we do with the `lib` package.